src/campaigns/campaigns.controller.ts

```diff
@@ -73,7 +73,7 @@ export class CampaignsController {
     @Body() body: UpdateCampaignSchema,
   ) {
     // TODO get campaign from req user
-    const updateResp = await this.campaignsService.update(id, body)
+    const updateResp = await this.campaignsService.updateJsonFields(id, body)
```

Comment:
If this method is only updating those few JSONb fields on the `campaign` table, then how do we update the other columns for a given `campaign`? Can we not reconcile this functionality into a singular `update` method on the service? 🤔 

Looks like we may be able to even branch this update logic based on the type of the Prisma model fields being updated:
<img width="1327" alt="Screenshot 2024-12-17 at 4 42 46 PM" src="https://github.com/user-attachments/assets/1a439200-2270-499e-b533-43b497fe0714" />